tests/stdlib/tparsesql.nim

```diff
@@ -207,3 +208,11 @@ SELECT `SELECT`, `FROM` as `GROUP` FROM `WHERE`;
 doAssert $parseSQL("""
 SELECT "SELECT", "FROM" as "GROUP" FROM "WHERE";
 """) == """select "SELECT", "FROM" as "GROUP" from "WHERE";"""
+
+doAssert $parseSQL("""CREATE TABLE Persons (Personid int NOT NULL AUTO_INCREMENT);""") == """create table Persons(Personid  int  not null  auto_increment );;"""
```

Comment:
```suggestion
check $parseSQL("""CREATE TABLE Persons (Personid int NOT NULL AUTO_INCREMENT);""") == """create table Persons(Personid  int  not null  auto_increment );;"""
```

ditto elsewhere (since you import unittest)